extern/acts/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ message( STATUS "Building Acts as part of the TRACCC project" )
 
 # Declare where to get Acts from.
 set( TRACCC_ACTS_SOURCE
-   "URL;https://github.com/krasznaa/acts/archive/refs/tags/v10.0.1.tar.gz;URL_MD5;fcd2f2f7b5810e34b8faa12634043be3"
+   "GIT_REPOSITORY;https://github.com/guilhermeAlmeida1/acts.git;GIT_TAG;patch-1"
```

Comment:
@krasznaa how should we go about handling this. I initially made it dependent on my own branch because it included a fix needed to acts. In the meantime this has been merged into the main branch (https://github.com/acts-project/acts/pull/1650 ).
It seems a bit unsafe to just change this dependency directly to the main branch of the Acts repository, but I don't think this PR has been added to any Acts release yet.

Reply:
See the last messages on Mattermost: https://mattermost.web.cern.ch/acts/channels/town-square

Reply:
Oh, this was some interesting timing with your comment here as well

Reply:
I'll hold onto this PR as a draft until we get the next release of Acts, then.